src/main/java/uk/gov/companieshouse/insolvency/data/model/InsolvencyDocument.java

```diff
@@ -20,17 +23,23 @@
 
     private final Updated updated;
```

Comment:
Shouldn't this be annotated with @Field?

Reply:
Code change done